.github/workflows/job_build_linux.yml

```diff
@@ -182,7 +182,7 @@ jobs:
       #
 
       - name: CMake configure - OpenVINO
-        run: eval cmake $CMAKE_OPTIONS -S ${OPENVINO_REPO} -B ${BUILD_DIR}
+        run: eval cmake ${{ env.CMAKE_OPTIONS }} -S ${OPENVINO_REPO} -B ${BUILD_DIR}
```

Comment:
optional, it would be better to align env usage then
```suggestion
        run: eval cmake ${{ env.CMAKE_OPTIONS }} -S ${{ env.OPENVINO_REPO }} -B ${{ env. BUILD_DIR }}
```